pkg/controller/performanceprofile/status.go

```diff
@@ -62,6 +63,12 @@ func (r *ReconcilePerformanceProfile) updateStatus(profile *performancev1.Perfor
 		modified = true
 	}
 
+	if profileCopy.Status.RuntimeClass == nil {
+		runtimeClassName := components.GetComponentName(profile.Name, components.ComponentNamePrefix)
+		profileCopy.Status.RuntimeClass = &runtimeClassName
```

Comment:
Can you explain more in the commit about why we set the RuntimeClass? It seems even a little explanation of "why" we set and what we do with it would be nice.

Reply:
done